pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -301,3 +301,15 @@ func (m *kubeGenericRuntimeManager) GetPortForward(podName, podNamespace string,
 	}
 	return url.Parse(resp.Url)
 }
+
+// DeleteSandbox removes the sandbox by sandboxID.
+func (m *kubeGenericRuntimeManager) DeleteSandbox(sandboxID string) error {
+	klog.V(4).Infof("Removing sandbox %q", sandboxID)
+	// stop sandbox is called as part of kill pod function but the error is ignored. So,
```

Comment:
comment got lost:

comment says the error is ignored, but the statement below actually returns an error. Is it intentional and I'm missing something in this comment?

Reply:
error from stopsandbox called as part of killpod function is ignored and hence we are calling stopsandbox again as part of delete sandbox.

Reply:
I understand this comment now. Thank you!